src/tests/api/test_subevents.py

```diff
@@ -50,6 +109,242 @@ def test_subevent_list(token_client, organizer, event, subevent):
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "presale_start": "2017-12-27T10:00:00Z",
+            "presale_end": "2017-12-26T10:00:00Z"
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"non_field_errors":["The event\'s presale cannot end before it starts."]}'
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+                meta_prop.name: "Conference"
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert organizer.events.get(slug=event.slug).subevents.get(id=resp.data['id']).meta_values.filter(
+        property__name=meta_prop.name, value="Conference"
+    ).exists()
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert not organizer.events.get(slug=event.slug).subevents.get(id=resp.data['id']).meta_values.filter(
+        property__name=meta_prop.name
+    ).exists()
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+                "test": "test"
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"meta_data":["Meta data property \'test\' does not exist."]}'
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "item_price_overrides": [
+                {
+                    "item": item.pk,
+                    "price": "99.99"
+                }
+            ],
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).items.get(id=item.pk).default_price == Decimal('23.00')
+    assert organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).item_price_overrides[item.pk] == Decimal('99.99')
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "item_price_overrides": [
+                {
+                    "item": 123,
+                    "price": "99.99"
+                }
+            ],
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"item_price_overrides":[{"item":["Invalid pk \\"123\\" - object does not exist."]}]}'
+
+
```

Comment:
I know it's annoying, but I'd like to request a few more tests there. I believe the item_price_override and variation_price_override are the most complex and most unique thing in this whole PR and I'd like to think it's worth testing that well. By that I mean that the tests should, for both parameters, cover all three major cases of

* a new price override being added in a PATCH command
* a price override being deleted in a PATCH command
* a price override being changed to a new price in a PATCH command

All of these could go wrong in different ways if we mess the logic up one day and it would be nice to notice that ;)